services/efs-guardian/requirements/_test.in

```diff
@@ -0,0 +1,32 @@
+#
+# Specifies dependencies required to run 'services/api-server/test'
+#  both for unit and integration tests!!
+#
+--constraint ../../../requirements/constraints.txt
+--constraint ./constraints.txt
+
+# Adds base AS CONSTRAINT specs, not requirement.
+#  - Resulting _text.txt is a frozen list of EXTRA packages for testing, besides _base.txt
+#
+--constraint _base.txt
+
+
+aiodocker
+asgi-lifespan
+coverage
+debugpy
+deepdiff
+docker
+faker
+fakeredis[lua]
+httpx
+moto[server]
+parse
+psutil
+pytest
+pytest-asyncio
+pytest-cov
+pytest-mock
+pytest-runner
+python-dotenv
+respx
```

Comment:
I think you are not currently using: `aidocker`, `asgi-lifespan`, `deepdiff`, `fakeredis`, `moto[server]`, `parse`, `psutil`.
Could you please remove them from the testing infrastructure? Also all the dependencies that are not used.
I much prefer that they are added when required, otherwise we end up with dependencies that block upgrades as PC states all the time.

Reply:
Yes, in next PR I am going to use all of them. But I will review it with the next functionality that is coming.

Reply:
The only reason I wrote this, is because I think some parts will be forgotten. We'll see how it goes. Currently there are unused dependencies.